sdcm/tester.py

```diff
@@ -856,7 +857,28 @@ def setUp(self):  # pylint: disable=too-many-branches,too-many-statements
                 db_cluster.validate_seeds_on_all_nodes()
                 validate_raft_on_nodes(nodes=db_cluster.nodes)
 
+    def _switch_to_network_replication_strategy(self, keyspaces: List[str]) -> None:
```

Comment:
This case is specific and I wouldn't generalize it - better create method like `switch_system_auth_to_network_topology_strategy` and focus on it.

Reply:
Just change logic of previous function